falcon/status_codes.py

```diff
@@ -90,8 +96,14 @@
 HTTP_EXPECTATION_FAILED = HTTP_417
 HTTP_418 = "418 I'm a teapot"
 HTTP_IM_A_TEAPOT = HTTP_418
+HTTP_421 = '421 Misdirected Request'
```

Comment:
I don't think we need this code, since it deals with connection reuse in HTTP/2, which is a web sever (not application) concern.